pkg/apis/internalversion/kwokctl_configuration_types.go

```diff
@@ -164,7 +163,8 @@ type KwokctlConfigurationOptions struct {
 	// KubeSchedulerPort is kube-scheduler port in the binary runtime
 	KubeSchedulerPort uint32
 
-	// KwokControllerPort is kube-controller port in the binary runtime
+	// KwokControllerPort is kwok-controller port that is exposed to the host.
+	// is the default value for flag --controller-port and env KWOK_CONTROLLER_PORT
```

Comment:
```suggestion
```

Reply:
Oh I see, I did not understand it well
Sorry :relaxed: Fixing it in a minute